### ISSUE TITLE
[Feature] Sidebery-like features and split-screen

### ISSUE THREAD

**Issue by Marin-Kitagawa** [brave/brave-browser#38407]
### Description

Sidebery has a lot of features. But there is no equivalent to Sidebery in Chrome. Below is the image of my Sidebery setup. 

![image](https://github.com/brave/brave-browser/assets/49131888/e30aa11b-de13-4850-91e6-f699c9559a44)

The left most icons denote the panels similar to Workspaces in Edge, Vivaldi, Arc, Opera and Floorp. The Sidebery sidebar completely hides similar to Arc browser (my customization). 

I'd like the following features to be brought into Brave

1. Workspaces
2. Nested Groups/Nested Folders similar to Arc & Sidebery
3. Manual and Automatic hibernation (unloading) of tabs, groups and Workspaces
4. Containers (as in Firefox)
5. Tab title customization
6. Clearing cookies per site
7. Split-screen view

Basically, all Sidebery features, Containers in Firefox and Split-screen view (split screen is supported by Arc, Vivaldi, Edge)

### Steps to reproduce

-

### Actual result

-

### Expected result

-

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

-

### Channel information

- [ ] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by sairkader** [comment]
Hello Marin, for some reason I cannot see anything in this section, I'd love to reproduce what you did with your sidebery. Do you have a discord or something else where I can message you on to ask how?

![Image](https://github.com/user-attachments/assets/5311c6b2-ef20-4946-9a76-c1ce8e08eb5d)

**Comment by Marin-Kitagawa** [comment]
@sairkader I apologize for the late response. Please see my repository here: [FirefoxConfig](https://github.com/Marin-Kitagawa/FirefoxConfig). It also contains a file called `sidebery.css` which is the CSS for Sidebery. One more thing to note is that the Sidebery panels will be on the left side instead of top as you can see in the image.

**Comment by Marin-Kitagawa** [comment]
@sairkader This would be an excellent time for adding these features to Brave especially after Firefox's new Terms of Use and Privacy Policy

**Comment by TheFuzzStone** [comment]
I agree with @Marin-Kitagawa.

After Mozilla/Firefox showed their true face, I immediately removed Firefox from all my devices.

But, Sidebery... has a stronger hold on me than I thought. And for now, I've switched to some fork of Firefox, simply because it has something that Brave doesn't currently have, namely:

* Sidebery
* web panel functionality
* ctrl-tab works more logically than in Chromium-based browsers.
